packages/ember-data/lib/serializers/json-serializer.js

```diff
@@ -988,6 +988,7 @@ var JSONSerializer = Serializer.extend({
     the payload and just sends the raw serialized JSON object.
     If your server expects namespaced keys, you should consider using the RESTSerializer.
     Otherwise you can override this method to customize how the record is added to the hash.
+    The hash property should be modified by reference (possibly using something like _.extend)
```

Comment:
I like this addition to the docs but I'd prefer to see "(possibly using something like _.extend)" removed. "possibly using something like" doesn't fit the "voice" of the docs.